src/components/Tabs/Tabs.jsx

```diff
@@ -1,44 +1,37 @@
 import cn from 'classnames';
 
-export const Tabs = ({ tabs, activeTab, onSelected }) => {
-  const test = tab => {
-    if (activeTab.id !== tab.id && tabs.find(({ id }) => id === activeTab.id)) {
-      onSelected(tab);
-    } else {
-      onSelected(tabs[0]);
-    }
-  };
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(({ id }) => id === activeTabId) || tabs[0];
 
   return (
-    <div className="section">
-      <h1 className="title">Selected tab is {activeTab.title}</h1>
-
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            {tabs.map(tab => {
-              return (
-                <li
-                  className={cn({ 'is-active': activeTab.id === tab.id })}
-                  data-cy="Tab"
-                  key={tab.id}
-                >
-                  <a
-                    href={`#${tab.id}`}
-                    data-cy="TabLink"
-                    onClick={() => test(tab)}
-                  >
-                    {tab.title}
-                  </a>
-                </li>
-              );
-            })}
-          </ul>
-        </div>
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(({ id, title }) => (
+            <li
+              data-cy="Tab"
+              key={id}
+              className={cn({ 'is-active': activeTab.id === id })}
+            >
+              <a
+                href={`#${id}`}
+                data-cy="TabLink"
+                onClick={(e) => {
+                  e.preventDefault();
+                  if (id !== activeTabId) {
+                    onTabSelected(id);
+                  }
+                }}
+              >
```

Comment:
Why did you delete a helper function?) put it back, just use more self-descriptive name
consider following [this](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) common naming convention